packages/mdc-dialog/foundation.ts

```diff
@@ -69,6 +69,7 @@ export class MDCDialogFoundation extends MDCFoundation<MDCDialogAdapter> {
   private scrimClickAction_ = strings.CLOSE_ACTION;
   private autoStackButtons_ = true;
   private areButtonsStacked_ = false;
+  private suppressedPressSelector: string = strings.SUPPRESS_DEFAULT_PRESS_SELECTOR;
```

Comment:
For consistency with the constant, let's rename this and the functions to 

- `suppressDefaultPressSelector`
- `getSupressDefaultPressSelector()`
- `setSupressDefaultPressSelector()`

Since we're suppressing the _default_ press and not all presses.

Reply:
ah i had misread it, i assumed DEFAULT meant its the _default selector_. rather than a selector for the default... 🤦